lib/registration.pm

```diff
@@ -205,8 +205,14 @@ sub fill_in_registration_data {
                 # yast would display empty pkg install screen if no addon selected on sle12 sp0
                 # set check_screen timeout longer to ensure the screen checked in this case
                 elsif (match_has_tag('yast-scc-emptypkg')) {
-                    send_key 'alt-a';
-                    last;    # The needle of empty packages has been update more specific, no need to enter determine statement again.
+                    wait_still_screen 5;
+                    if (check_screen('yast-scc-emptypkg')) {
```

Comment:
Please try to avoid `check_screen` with non-zero timeout in general and use assert_screen with multiple tags and use 'match_has_tag'. In this specific case here 'match_has_tag' has been already used in before so that does not make any sense to me. I don't understand what problem you are trying to solve here. Maybe the needle itself is too generic and matches when it shouldn't?